jabgui/src/main/java/org/jabref/gui/libraryproperties/general/GeneralPropertiesViewModel.java

```diff
@@ -196,23 +195,24 @@ public StringProperty laTexFileDirectoryProperty() {
     }
 
     private Path getBrowseDirectory(String configuredDir) {
+        Path libPath = preferences.getFilePreferences().getWorkingDirectory();
```

Comment:
Strange.

I think, this really should be `org.jabref.model.database.BibDatabaseContext#getDatabasePath`

Reason: If having 10 library files opened, the working directory is not the one of the current file.